src/dpi.erl

```diff
@@ -85,6 +100,13 @@ load_unsafe() ->
 %   local helper functions
 %===============================================================================
 
+reg(SlaveNode, Pid) ->
+    Name = {?MODULE, SlaveNode, make_ref()},
```

Comment:
Adding `node()` to the Name will give information from which this reference was created.
`Name = {?MODULE, SlaveNode, node(), make_ref()},`
This should make is easier to monitor the distribution of the slave nodes